documentation/README.md

```diff
@@ -24,8 +24,8 @@ When running a dotnet application differences in diverse local and production en
     - [JSON Schema](./schema.json)
 - [Authentication](./authentication.md)
     - [API Key (Recommended)](./authentication.md#api-key-authentication)
-    - [Windows](./authentication.md#windows-authentication)    
-- [Egress Providers](#)
-- [Troubleshooting](#)
+    - [Windows](./authentication.md#windows-authentication)
+- [Egress Providers](./egress.md)
+- [Troubleshooting](./troubleshooting.md)
```

Comment:
I didn't mean to merge troubleshooting.md yet, but i'll keep it for now unless someone is already adding content.